### DISCUSSION TITLE
Related to new portman version

### DISCUSSION THREAD

**Discussion by shubhambajad** [apideck-libraries/portman#389] [General]
Hi Team,

When you are planning to release new Portman version which includes content-type related changes and other things as well?

Also, if it is release then is it documented so we can try those new things?

**Comment by thim81** [comment]
Hi @shubhambajad 

We released 2 new versions (end June, mid of July) which included various new features, including improvements towards the content-type detection & handling

See the changelog for more details
https://github.com/apideck-libraries/portman/blob/main/CHANGELOG.md
https://github.com/apideck-libraries/portman/releases/tag/v1.18.0

What are you still missing?

**Reply by shubhambajad** [reply]
@thim81 Thanks team, I haven't checked this doc so I missed it. I will update my framework according to that.
Also, is there any chance for overwrite http method feature?
